### ISSUE TITLE
macOS: changes to fallout.cfg and f1_res.ini have no effect

### ISSUE THREAD

**Issue by strelich** [alexbatalov/fallout1-ce#155]
I've installed fallout1-ce on my M2 Mac running macOS 13.6.6, using game files extracted from the GOG download. I'm able to launch and run the game just fine (made it to the radscorpion cave) but with no music. I've modified `fallout.cfg` as suggested elsewhere (e.g., so that the file names/paths are properly capitalized), but this does not appear to have any effect on the game (i.e., it ran fine without changing `master.dat` to `MASTER.DAT`, and there's still no music after changing the music path to `DATA/SOUND/MUSIC/`. Similarly, nothing I change in `f1_res.ini` has any effect on the game's resolution, windowed mode, etc., except that some changes made the game stop recognizing mouse clicks (mouse movement and key presses still worked fine). Any suggestions would be appreciated!

**Comment by nahweijie1** [comment]
I am running it on an M1 Mac with macOS 14.1.1. Everything is working fine, from music to resolution changes.

Firstly, can you double-check that the file extensions are capitalized in your data/sound/music folder? (i.e. files ending with .acm should be renamed to .ACM)

Additionally, could you double-check if the .cfg and .ini files you edited are in the same directory as your Fallout CE app? As far as I know, there seems to be a redundant app folder.

![image](https://github.com/alexbatalov/fallout1-ce/assets/93656280/1404357e-66e8-4b53-881e-c6cf838c4a7b)

**Comment by strelich** [comment]
Yes, the file names/extensions in the music folder are capitalized (e.g., `01HUB.ACM`). The `fallout.cfg` and `f1_res.ini` files I've been editing are in the same `Fallout` directory as the Fallout CE app. I didn't see duplicates of them in `Fallout/app/`, but there are duplicates in `Fallout/__support/app`; editing these has no effect either.

I notice that on GOG there are two installers listed for Fallout (Part 1 and Part 2). I only used the first one (`setup_fallout_1.2_(27130).exe`), via the Extractor app -- what should be done with Part 2 (`setup_fallout_1.2_(27130)-1.bin`)?

**Comment by jcfrancisco** [comment]
@strelich I was able to fix this by copying `/Applications/Fallout/__support/app/f1_res.ini` to `/Applications/Fallout/f1_res.ini` and editing that.

**Comment by strelich** [comment]
I deleted `Fallout/__support/app/f1_res.ini` and `Fallout/__support/app/fallout.cfg`, and fallout-ce now appears to be correctly reading `Fallout/f1_res.ini` and `Fallout/fallout.cfg` (i.e., changes to these files affect the game, music plays, &c)

**Comment by AlmightyBeaver** [comment]
I did all of the above, but for me Fallout/f1_res.ini is completely ignored and no music in the game.
